### PR TITLE
Replace gogits/git-module dependency with go-gitea/git

### LINKED ISSUE #92: Replace gogits git-module dependency
We want to replace the current dependency to `gogits/git-module` with `go-gitea/git`, below you can see a list of greped values.

```
# grep -R git-module *
cmd/serve.go:	git "github.com/gogits/git-module"
models/action.go:	"github.com/gogits/git-module"
models/git_diff.go:	"github.com/gogits/git-module"
models/git_diff.go:// TODO: move this function to gogits/git-module
models/git_diff.go:// TODO: move this function to gogits/git-module
models/pull.go:	"github.com/gogits/git-module"
models/release.go:	"github.com/gogits/git-module"
models/repo.go:	git "github.com/gogits/git-module"
models/repo_branch.go:	"github.com/gogits/git-module"
models/repo_editor.go:	git "github.com/gogits/git-module"
models/update.go:	git "github.com/gogits/git-module"
models/user.go:	"github.com/gogits/git-module"
models/webhook_slack.go:	"github.com/gogits/git-module"
models/wiki.go:	"github.com/gogits/git-module"
modules/context/repo.go:	"github.com/gogits/git-module"
routers/api/v1/convert/convert.go:	"github.com/gogits/git-module"
routers/api/v1/repo/file.go:	"github.com/gogits/git-module"
routers/install.go:	"github.com/gogits/git-module"
routers/repo/commit.go:	"github.com/gogits/git-module"
routers/repo/download.go:	"github.com/gogits/git-module"
routers/repo/editor.go:	"github.com/gogits/git-module"
routers/repo/http.go:	git "github.com/gogits/git-module"
routers/repo/middlewares.go:	"github.com/gogits/git-module"
routers/repo/pull.go:	"github.com/gogits/git-module"
routers/repo/repo.go:	"github.com/gogits/git-module"
routers/repo/setting.go:	"github.com/gogits/git-module"
routers/repo/view.go:	"github.com/gogits/git-module"
routers/repo/webhook.go:	git "github.com/gogits/git-module"
routers/repo/wiki.go:	"github.com/gogits/git-module"
vendor/vendor.json:			"path": "github.com/gogits/git-module",
```

### PR DESCRIPTION
**Pull request by strk**
Fixes #92

### CODE REVIEW

File: cmd/serve.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 	"time"
 
 	"github.com/Unknwon/com"
-	git "github.com/gogits/git-module"
+	git "github.com/go-gitea/git"
```
Comment (tboerger) [inline cmd/serve.go]: Please drop this git alias everywhere, not needed anymore.

**Review by tboerger — CHANGES_REQUESTED** [review]

**Comment by strk** [comment]
On Sun, Nov 06, 2016 at 02:35:40AM -0800, Thomas Boerger wrote:

> Please drop this git alias everywhere, not needed anymore.

Done.

**Comment by andreynering** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**